plugins/fun/fun.py

```diff
@@ -50,3 +59,8 @@ def get_memes(self):
         image = response["url"]
         return image
 
+    def get_weather(self):
+        response = requests.get(self.WEATHER_API).json()
+        status = response["url"]
```

Comment:
@RonnieTim double check that this is `response["url"]` from the response I got back it looks like it might be `response["message"]`